<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CookiesWidget</class>
 <widget class="QWidget" name="CookiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>478</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="search">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="EditTableView" name="cookiesTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAllButton">
       <property name="text">
        <string>Remove &amp;All Cookies</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EditTableView</class>
   <extends>QTableView</extends>
   <header>edittableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
